core/services/workflows/v2/capability_executor.go

```diff
@@ -86,3 +87,7 @@ func (c *CapabilityExecutor) CallCapability(ctx context.Context, request *sdkpb.
 		},
 	}, nil
 }
+
+func (c *ExecutionHelper) GetId() string {
```

Comment:
The linter wants idiomatic casing here:
```suggestion
func (c *ExecutionHelper) GetID() string {
```

Reply:
That's annoying. We should set up common to have the same failures so this doesn't block core.

Reply:
It should have the same. What do you think it missed?

Reply:
A required test shouldn't fail in core because of an interface's method name in common. Common should have had the required test.

Also, what would I have done if the library wasn't one that we owned?

Reply:
[PR](https://github.com/smartcontractkit/chainlink-common/pull/1248/commits) to fix the name